.chloggen/3330-python-otel-logs-exporter.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp, github action)
+component: auto-instrumentation
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: set OTEL_LOGS_EXPORTER env var to oltp in python instrumentation
```

Comment:
```suggestion
note: set OTEL_LOGS_EXPORTER env var to oltp in python instrumentation to avoid setting it by the user if they decide to enable logs
```